ocs_ci/ocs/platform_nodes.py

```diff
@@ -1907,6 +1907,29 @@ def attach_nodes_to_cluster(self, node_list):
             "attach nodes to cluster functionality is not implemented"
         )
 
+    def restart_nodes_by_stop_and_start_teardown(self):
+        """
+        Make sure all VM instances up by the end of the test
+
+        """
+        self.cluster_nodes = get_node_objs()
+        vms = self.get_vm_names()
```

Comment:
Shouldn't this be  `vms = self.azure.get_vm_names()` ?

Reply:
Good catch. updated.